src/sagemaker/model.py

```diff
@@ -345,6 +375,16 @@ def compile(
                 model. Allowed values: 'mxnet', 'tensorflow', 'keras', 'pytorch',
                 'onnx', 'xgboost'
             framework_version (str):
+            target_platform_os (str): Target Platform OS. Allowed values: 'LINUX', 'ANDROID'.
+                It can be used instead of target_instance_family.
+            target_platform_arch (str): Target Platform Architecture. Allowed values: 'X86_64',
+                'X86', 'ARM64', 'ARM_EABIHF', 'ARM_EABI'.
+                It can be used instead of target_instance_family.
+            target_platform_accelerator (str): Target Platform Accelerator. Allowed values:
+                'INTEL_GRAPHICS', 'MALI', 'NVIDIA'. Optional.
+            compiler_options (str): Additional parameters for compiler (Optional, in JSON format).
```

Comment:
if it needs to be in JSON, would it make more sense to accept a dict, and then let the Python SDK call `json.dumps`?

Reply:
The API reference is updated.
[OutputConfig API](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_OutputConfig.html)
[TargetPlatform API](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_TargetPlatform.html)

Reply:
Fixed, now it support both `dict` and `string`. Similar to `DataInputConfig`